lib/rouge/lexers/ecl.rb

```diff
@@ -0,0 +1,140 @@
+# -*- codding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+  module Lexers
+    class ECL < RegexLexer
+      tag 'ecl'
+      filenames '*.ecl'
+      mimetypes 'application/x-ecl'
+
+      title "ECL"
+      desc "Enterprise Control Language"
```

Comment:
Is there a URL that can be included in the description for reference (see [the Rust lexer](https://github.com/rouge-ruby/rouge/blob/22ac67833d29a1996ba4b3d3f62577a15eabec2d/lib/rouge/lexers/rust.rb), for example).

Reply:
Added url